### PR TITLE
:book: Add reference documentation for Nvidia ARM boards

### LINKED ISSUE #45: Feature Request: support NVIDIA Jetson Nano
**Is your feature request related to a problem? Please describe.**
Support the [NVIDIA Jetson Nano](https://developer.nvidia.com/embedded/jetson-nano-developer-kit) device. More and more enterprises are building AI/inferencing apps using Jetson appliances.

**Describe the solution you'd like**
N/A

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #793: investigate nvidia ARM support

### PR DESCRIPTION
**Pull request by mudler**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Closes #45 
Fixes #793

### CODE REVIEW

**Comment by mudler** [comment]
I've also pushed a `development/jetson` branch for testing/development

File: docs/content/en/docs/Development/nvidia.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+date: 2022-11-13
+description: >
+    This page contains a reference on how to run Kairos on Nvidia Jetson ARM
+---
+
+{{% alert title="Note" %}}
+Please note that the following page contains only development reference. At the time of writing, we have tried porting Kairos to Jetson Nano eMMC without success. This is due to the old kernel supported (4.9), not properly working with `EFISTUB` and `U-boot` (you can see the [issue here](https://github.com/kairos-io/kairos/issues/45)). However, the steps outlined _should_ be a good reference to port Kairos to those architecture _when_ a new kernel version is available. We have tested, and have successfully booted a Jetson Nano with the 5.15 kernel, however, due to the lack of driver support, eMMC partitions are not properly recognized.
+{{% /alert %}}
+
+This page is a development reference in order to boot Kairos in Nvidia Jetson devices. Nvidia Jetson images by default ship `extlinux` as bootloader, without EFI boot. This guide explains how to get instead u-boot to chainload to `grub2`, which can be used to boot and load `Kairos`.
```
Comment (jimmykarily) [inline docs/content/en/docs/Development/nvidia.md:14]:
```suggestion
This page is a development reference in order to boot Kairos in Nvidia Jetson devices. Nvidia Jetson images by default ship `extlinux` as bootloader, without EFI boot. This guide explains how to get u-boot to chainload to `grub2`, which can be used to boot and load `Kairos`.
```

File: docs/content/en/docs/Development/nvidia.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+Note that currently there are no official Kairos core images for Jetson images, this page will refer to Jetson Nano eMMC version as the current reference, but the steps should be similar, as outline how to use the Nvidia SDK to flash the OS onboard in the eMMC of the device.
```
Comment (jimmykarily) [inline docs/content/en/docs/Development/nvidia.md:16]:
```suggestion
Note that currently there are no official Kairos core images for Jetson images. This page will refer to Jetson Nano eMMC version as the current reference, but the steps should be similar, as outline how to use the Nvidia SDK to flash the OS onboard in the eMMC of the device.
```

Comment (jimmykarily) [reply docs/content/en/docs/Development/nvidia.md:16]: Which steps are "similar" to what? This sentence implies there is some other version but it's not implicitly mentioned.

File: docs/content/en/docs/Development/nvidia.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+The steps involved are:
+
+- Prepare the Kernel (if you have one, compatible with `EFISTUB`, you can skip this part)
+- Flash u-boot (If the U-boot version support booting efi shells, you might skip this part too)
+- Prepare the Kairos partitions
+- Flash the image to the board
+
+## Prerequisites
+
+You need the Nvidia SDK and few other dependencies in the system. Note that for the Jetson Nano you can't use the latest SDK version as it is not anymore supporting it. The latest version available with support for Jetson Nano is [r32.7.3](https://developer.nvidia.com/embedded/linux-tegra-r3273):
```
Comment (jimmykarily) [inline docs/content/en/docs/Development/nvidia.md:27]:
```suggestion
You need the Nvidia SDK and few other dependencies in the system. Note that Jetson Nano is no longer supported by the SDK. The latest version available with support for Jetson Nano is [r32.7.3](https://developer.nvidia.com/embedded/linux-tegra-r3273):
```

**Review by jimmykarily — APPROVED** [review]
:four_leaf_clover:  Add  "- good luck" in the prerequisites :D 

It's good to have this as a start though, good job!

**Pull request merged**